clang/test/ParserOpenACC/parse-cache-construct.cpp

```diff
@@ -74,12 +74,12 @@ void use() {
   for (int i = 0; i < 10; ++i) {
     // FIXME: Once we have a new array-section type to represent OpenACC as
     // well, change this error message.
```

Comment:
This FIXME was resolved by this commit, consider removing it in a trivial follow-up change.

Reply:
Thanks, done in bb1a8bbb